libs/java/service/src/main/java/ca/bc/gov/tno/services/kafka/config/KafkaProducerConfig.java

```diff
@@ -0,0 +1,51 @@
+package ca.bc.gov.tno.services.kafka.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Configuration settings for Kafka producer.
+ */
+@Configuration
+public class KafkaProducerConfig extends KafkaConfig {
```

Comment:
I'd love to know how to make optional @Configuration beans in java.  Presently I have to make all @Value properties optional so that they do not throwing errors in services that don't need them.